packages/apps/job-launcher/server/src/common/config/env-schema.ts

```diff
@@ -28,6 +28,7 @@ export const envValidator = Joi.object({
   WEB3_PRIVATE_KEY: Joi.string().required(),
   GAS_PRICE_MULTIPLIER: Joi.number(),
   REPUTATION_ORACLE_ADDRESS: Joi.string().required(),
+  REPUTATION_ORACLE_URI: Joi.string().required(),
```

Comment:
Why having it in .env when this info is available in KVStore?

Reply:
Yes, agree with you, let me fix it